mitmproxy/tools/web/app.py

```diff
@@ -221,6 +220,10 @@ def __init_subclass__(cls, **kwargs):
             if fn is not tornado.web.RequestHandler._unimplemented_method:
                 setattr(cls, method, AuthRequestHandler._require_auth(fn))
 
+    def get_auth_cookie_name(self):
```

Comment:
```suggestion
    @staticmethod
    def auth_cookie_name():
```
(and then reuse in the backend)